enterprise/server/registry/registry_server.go

```diff
@@ -53,6 +53,16 @@ func (t *RegistryServer) Start() {
 			return
 		}
 
+		if len(pathPaths) > 3 && pathPaths[1] == "modules" && strings.HasPrefix(pathPaths[3], "npm") {
+			bytes, status, err := handleNPM(path)
```

Comment:
s/bytes/buf/g

Reply:
b/c bytes is a commonly used package already

Reply:
Done